django/contrib/admin/static/admin/css/base.css

```diff
@@ -24,7 +24,7 @@ html[data-theme="light"],
     --breadcrumbs-link-fg: var(--body-bg);
     --breadcrumbs-bg: var(--primary);
 
-    --link-fg: #447e9b;
+    --link-fg: #437c98;
```

Comment:
The new color have enough contrast when the background is white. Unfortunately `--link-fg` is also used with yellow (`#ffffcc`) and gray backgrounds (`#f8f8f8`). We should make it even darker to meet AA standards, e.g. `#417893` (\cc @knyghty.)